src/main/java/dev/drugowick/timeseriespoc/controller/NewMeasurementController.java

```diff
@@ -3,14 +3,14 @@
 import dev.drugowick.timeseriespoc.controller.dto.MeasurementInput;
 import dev.drugowick.timeseriespoc.domain.entity.Measurement;
 import dev.drugowick.timeseriespoc.service.UserDataService;
+import jakarta.validation.Valid;
```

Comment:
Change to `jakarta` namespace (under Eclipse something IIRC) (from `javax` - which was under Oracle, I guess)